src/main/java/com/reckue/post/controllers/CommentController.java

```diff
@@ -30,27 +33,45 @@
 
     /**
      * This type of request allows to create and process it using the converter.
+     * Throws {@link ReckueUnauthorizedException} in case if token is absent.
      *
      * @param commentRequest the object of class CommentRequest
      * @return the object of class CommentResponse
      */
     @PostMapping
-    public CommentResponse create(@RequestBody @Valid CommentRequest commentRequest) {
-        return convert(commentService.create(convert(commentRequest)));
+    // TODO: add postId as PathVariable and delete it from request (you don't need to enter a postId in update method)
+    public CommentResponse create(@RequestBody @Valid CommentRequest commentRequest,
+                                  HttpServletRequest request) {
```

Comment:
You need to take it to a higher level, that is before controller level.

Reply:
We can do it in the nearest future.